File: exampleNET/exampleNET/Http.cs
using System.Net.Http;
using System.Threading.Tasks;

namespace exampleNET
{
    public class Http
    {
        private readonly string _baseUrl = "https://jsonplaceholder.typicode.com/";

        public async Task<string> GetData(string path)
        {
            HttpResponseMessage response = await new HttpClient().GetAsync(_baseUrl + path);
            return await response.Content.ReadAsStringAsync();
        }
    }
}

File: exampleJAVA/src/main/java/Http.java
import org.springframework.web.client.RestTemplate;

public class Http {

    private static final String BASE_URL = "https://jsonplaceholder.typicode.com/";

    public String getData(String path) {
        return new RestTemplate().getForEntity(BASE_URL + path, String.class).getBody();
    }
}
